src/components/markdown.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import propTypes from 'prop-types';
+import marksy from 'marksy';
+import mdxComponentMap from '../utils/mdx-component-mapper';
+import indentNormalizer from '../utils/indent-normalizer';
+import { CodePane, Slide } from '../index';
+
+const _CodePane = ({ language, code }) => (
+  <CodePane language={language}>{code}</CodePane>
+);
+
+const compile = marksy({
+  createElement: React.createElement,
+  elements: {
+    ...mdxComponentMap,
+    codeblock: _CodePane,
+    code: _CodePane
+  }
+});
```

Comment:
The marksy component mapping is a near 1:1 with out MDX except for Code Panes which just require us to map the `code` prop to `children` as above.